packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -84,6 +84,7 @@ import { ArgumentProcessor } from '../plugin/command-registry';
 import { MaybePromise } from '@theia/core/lib/common/types';
 import { QuickTitleButton } from '@theia/core/lib/common/quick-open-model';
 import * as files from '@theia/filesystem/lib/common/files';
+import { FileStat } from '@theia/filesystem/lib/common/files';
```

Comment:
Can be removed, and line 556 can be updated to:

```ts
    $getWorkspaceFile(): Promise<files.FileStat | undefined>;
```

Reply:
totally right !, thanks !